Specs/Scene/ModelExperimental/ModelExperimentalSpec.js

```diff
@@ -205,6 +208,26 @@ describe(
       });
     });
 
+    it("gets credits from gltf", function () {
+      const resource = Resource.createIfNeeded(boxTexturedGltfWithCreditsUrl);
+      return resource.fetchJson().then(function (gltf) {
+        return loadAndZoomToModelExperimental(
+          {
+            gltf: gltf,
+            basePath: boxTexturedGltfWithCreditsUrl,
+          },
+          scene
+        ).then(function (model) {
+          const asset = model._sceneGraph.components.asset;
+          const credits = asset.copyright;
+          expect(credits.length).toBe(3);
+          credits.forEach(function (credit) {
+            expect(credit).toBeInstanceOf(Credit);
+          });
```

Comment:
The more important thing to check is that credits actually get pushed. I see this was modeled after `ModelSpec.js` but it can be more robust here.

Maybe something like:

```js
scene.renderForSpecs();
const credits = scene.frameState.creditDisplay._currentFrameCredits.screenCredits;
... check that the credits exist
```